includes/functions/functions_email.php

```diff
@@ -281,46 +287,44 @@ function zen_mail($to_name, $to_address, $email_subject, $email_text, $from_emai
                         $mail->Password = trim($email_password);
                     }
                     break;
                 case 'smtpauth':
                     $mail->isSMTP();
                     $mail->SMTPAuth = true;
                     $mail->Username = (!empty(trim($email_mailbox))) ? trim($email_mailbox) : EMAIL_FROM;
-                    if (trim($email_password) != '') {
+                    if (trim($email_password) !== '') {
                         $mail->Password = trim($email_password);
                     }
-                    $mail->Host = (trim($email_mail_server) != '') ? trim($email_mail_server) : 'localhost';
-                    if ((int)$email_mail_server_port != 25 && (int)$email_mail_server_port != 0) {
+                    $mail->Host = (trim($email_mail_server) !== '') ? trim($email_mail_server) : 'localhost';
+                    if ((int)$email_mail_server_port !== 25 && (int)$email_mail_server_port !== 0) {
                         $mail->Port = (int)$email_mail_server_port;
                     }
-                    if ((int)$mail->Port < 30 && $mail->Host == 'smtp.gmail.com') {
+                    if ((int)$mail->Port < 30 && $mail->Host === 'smtp.gmail.com') {
                         $mail->Port = 587;
                     }
                     //set encryption protocol to allow support for secured email protocols
-                    if ($mail->Port == '465') {
+                    if ($mail->Port === 465) {
                         $mail->SMTPSecure = PHPMailer::ENCRYPTION_SMTPS;
                     }
-                    if ($mail->Port == '587') {
+                    if ($mail->Port === 587) {
                         $mail->SMTPSecure = PHPMailer::ENCRYPTION_STARTTLS;
                     }
 
                     if (!$sending_newsletter) {
-                        if (defined('SMTPAUTH_EMAIL_PROTOCOL') && SMTPAUTH_EMAIL_PROTOCOL != 'none') {
+                        if (defined('SMTPAUTH_EMAIL_PROTOCOL') && SMTPAUTH_EMAIL_PROTOCOL !== 'none') {
                             $mail->SMTPSecure = SMTPAUTH_EMAIL_PROTOCOL;
                         }
-                    } else {
-                        if (defined('NEWSLETTER_SMTPAUTH_EMAIL_PROTOCOL') && NEWSLETTER_SMTPAUTH_EMAIL_PROTOCOL != 'none') {
-                            $mail->SMTPSecure = NEWSLETTER_SMTPAUTH_EMAIL_PROTOCOL;
-                        }
+                    } elseif (defined('NEWSLETTER_SMTPAUTH_EMAIL_PROTOCOL') && NEWSLETTER_SMTPAUTH_EMAIL_PROTOCOL !== 'none') {
+                        $mail->SMTPSecure = NEWSLETTER_SMTPAUTH_EMAIL_PROTOCOL;
                     }
                     break;
                 case 'smtp':
                     $mail->isSMTP();
                     $mail->Host = trim($email_mail_server);
-                    if ((int)$email_mail_server_port != 25 && (int)$email_mail_server_port != 0) {
+                    if ((int)$email_mail_server_port !== 25 && (int)$email_mail_server_port !== 0) {
```

Comment:
These int casts shouldn't be needed - we have converted when we created this variable.

Reply:
Removed in latest push.